README.md

```diff
@@ -1,2 +1,13 @@
 # privacy-security
-Privacy &amp; Security Principles, Documents and Testing
+
+This repository is for maintaining...
+
+* Principles relating to security and privacy.  A list of concrete, technical, verifiable privacy principles and tests that developers should build into Safe Paths and Safe Places, and testers should verify.
+* Test reports relating to Security and Privacy
```

Comment:
External test reports related to our adherence to security and privacy (i.e. static code analysis).